pkg/gitproviders/common.go

```diff
@@ -4,15 +4,22 @@ import (
 	"context"
 	"errors"
 	"fmt"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/weaveworks/weave-gitops/pkg/utils"
 
 	"github.com/fluxcd/go-git-providers/github"
 	"github.com/fluxcd/go-git-providers/gitprovider"
 	"github.com/weaveworks/weave-gitops/pkg/override"
 )
 
+type ProviderAccountType string
+
 const (
-	OwnerTypeUser = "user"
-	OwnerTypeOrg  = "organization"
+	UserAccountType ProviderAccountType = "user"
```

Comment:
One small change here, it's common in go to prefix the constants that are "kinda enums", so instead of `UserAccountType` and `OrgAccountType`, it would be `AccountTypeUser` and `AccountTypeOrg`.

you can see something like that in the HTTP package: https://golang.org/pkg/net/http/#pkg-constants

Reply:
It seemed easier to have the User and Org as prefix for IDE autocomplete 🤔